CHANGELOG.md

```diff
@@ -206,7 +206,7 @@ The following are deprecated and will be removed in a future release. For anythi
 <details>
   <summary>Click to expand the complete list</summary>  
 
-- Compatibility with implementations that don't support async/await at runtime, within AMD modules, is deprecated since version 4.25. For example, Angular applications using esri-loader will need to migrate from AMD modules to using @arcgis/core ES modules.
+- Compatibility with ES5 class utilities is deprecated for AMD builds, and will be removed at 4.28. For example, this will remove the ability to use Dojo to interact with the SDKs classes using Dojo's `declare` and `lang` modules.
```

Comment:
**question**: would it be possible to add a code example of what will no longer work rather than the text example. I don't follow the compatibility. It might be more of a if you use it you'll know?

Reply:
Perhaps a link to here? https://dojotoolkit.org/reference-guide/1.10/dojo/_base/declare.html#inheritance

Reply:
@steveoh, for example this wouldn't work anymore:

```js
        lang.extend(Map /* esri/Map */, {
           myNewProperty: false 
        });
```